docs/overview/cmake-can-not.rst

```diff
@@ -119,30 +119,27 @@ platform independently.
 Unrelocatable projects
 ======================
 
-Internally :ref:`CMake <CMake>` save full paths to the sources so it's not
-possible to generate project and then share it between several developers.
-It means you can't be CMake-guy who will generate different projects for
-Xcode-guys and VisualStudio-guys. All developers in team at least should be
-aware of how to generate projects using CMake. On practice it means they have
-to know CMake arguments to use, literally it's
-``cmake -H. -B_builds -GXcode``/``cmake -H. -B_builds "-GVisual Studio 12 2013"``,
-plus remember workflow notes from the section above.
-Next logic can be applied here: if you're working with developer then developer
-should learn tools you've used, if you're working with users then it's your
-responsibility to create user-friendly installers like ``*.msi`` instead of
-projects.
+Internally, :ref:`CMake <CMake>` saves the full paths to each of the sources,
+so it's not possible to generate a project then share it between several developers.
+In other words, you can't be "the CMake person" who will generate seperate projects for
+those who use Xcode and those who use Visual Studio. All developers in the team should be
+aware of how to generate projects using CMake. In practice it means they have
+to know which CMake arguments to use, some basic examples being
+``cmake -H. -B_builds -GXcode`` and ``cmake -H. -B_builds "-GVisual Studio 12 2013"``
+for Xcode and Visual Studio, respectively. Additionaly, they must understand the 
+[changes they must make in their workflow](https://cgold.readthedocs.io/en/latest/overview/cmake-can-not.html#affecting-workflow).
```

Comment:
It's not always must be `cgold.readthedocs.io`, e.g. if you build documentation locally it should be reference to your local documentation. Just using `:ref:` will do the trick.

Reply:
:ref: is a Sphinx keyword, not a Git one. So while it works with ReadTheDocs, it does not work natively on Git. Nevertheless, I changed it to be a :ref: .

Reply:
> :ref: is a Sphinx keyword, not a Git one. So while it works with ReadTheDocs, it does not work natively on Git. Nevertheless, I changed it to be a :ref:

You mean GitHub and how it renders RST?